ui/app/src/App.tsx

```diff
@@ -28,14 +29,16 @@ function App() {
   }
 
   return (
-    <PluginRegistry
-      loadingFallback="Loading..."
-      ErrorFallbackComponent={AlertErrorFallback}
-    >
-      <DataSourceRegistry>
-        <DashboardView resource={dashboard} />
-      </DataSourceRegistry>
-    </PluginRegistry>
+    <CoreView>
```

Comment:
I think `<Layout>` or so would be a more common naming for the overall page layout template?

Reply:
`Layout` could be in conflict with the dasboard code / datamodel as you have this notion as well :(.
Maybe another name ?

Reply:
Yeah, I typically start the same way (`Header` and `Footer` directly in the root `App` component), so if you wanted to do it that way, totally cool.